.github/workflows/deploy.yml

```
@@ -29,8 +29,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-ruby@v1
+      - uses: ruby/setup-ruby@v1
       - uses: actions/setup-node@v2
+        with:
+          ruby-version: 3.0.2
+          bundler-cache: true
```

We should specify `ruby-version` and `bundler-cache` to `ruby/setup-ruby` not `actions/setup-node`:

```suggestion
      - uses: ruby/setup-ruby@v1
        with:
          ruby-version: "3.0"
          bundler-cache: true
      - uses: actions/setup-node@v2
```

And we can use `"3.0"` instead of `3.0.2`. `"3.0"` can use the latest Ruby 3.0.X automatically. We don't need to care minor version in this repository.